### PR TITLE
[MCG] Add support for deploying NooBaa operator SCC along with Rook SCCs

### PR DESCRIPTION
**Pull request by tangledbytes**
This PR adds support for provisioning SCC for NooBaa operator alongside the Rook Operator. NooBaa operator added this SCC due to the IBM security requirements raised a few months ago.

### CODE REVIEW

**Comment by tangledbytes** [comment]
Created this PR as part of IBM security fixes requirement for NooBaa/MCG.

/cc @malayparida2000 @iamniting

File: controllers/storagecluster/noobaa_system_reconciler.go [hunk 1]
```diff
@@ -157,7 +157,7 @@ func (r *StorageClusterReconciler) setNooBaaDesiredState(nb *nbv1.NooBaa, sc *oc
 
 	nb.Spec.Tolerations = placement.Tolerations
 	nb.Spec.Affinity = &corev1.Affinity{NodeAffinity: placement.NodeAffinity}
-	nb.Spec.DBVolumeResources = &dBVolumeResources
+	nb.Spec.DBVolumeResources = convertCoreResourcesToVolumeResources(&dBVolumeResources)
```
Comment (iamniting) [inline controllers/storagecluster/noobaa_system_reconciler.go:160]:
```suggestion
	nb.Spec.DBVolumeResources = &corev1.VolumeResourceRequirements{
		Limits:   dBVolumeResources.Limits,
		Requests: dBVolumeResources.Requests,
	}
```

**Review by iamniting — CHANGES_REQUESTED** [review]
Pls, remove the extra signed-off line from the commit msg.

**Review by iamniting — APPROVED** [review]

**Pull request merged**